fs/operations/operations.go

```diff
@@ -2165,3 +2166,28 @@ func GetFsInfo(f fs.Fs) *FsInfo {
 	}
 	return info
 }
+
+var interactiveMutex sync.Mutex
+
+func skipDesctructive(ctx context.Context, action string) (string, bool) {
+	var (
+		flag string
+		skip bool
+	)
+	switch {
+	case fs.Config.DryRun:
+		flag = "--dry-run"
+		skip = true
+	case fs.Config.Interactive:
+		flag = "--interactive"
+		interactiveMutex.Lock()
+		defer interactiveMutex.Unlock()
+		skip = !config.ChooseAnswer(
+			fmt.Sprintf("rclone will %s. Are you sure? y/n\n", action),
+		)
+	default:
+		skip = false
+	}
+	message := fmt.Sprintf("Not %s as %s is set", action, flag)
```

Comment:
This function should probably just do the logging rather than returning the message and just return a bool